swfo/brdf_fallback.py

```diff
@@ -858,31 +831,41 @@ def write_brdf_fallback(brdf_dir, outdir, tile, year_from, year_to, filter_size,
     
     # remove doy 366 from further processing, final results for 366 will be symlinked to doy 365
     set_doys.remove(366)
```

Comment:
what if `set_doys` does not have 366? can I suggest `set_doys.pop(366, None)`?

Reply:
Yeah, that's a good idea